ui/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessRepository.scala

```diff
@@ -119,11 +119,11 @@ class DBProcessRepository(val dbConfig: DbConfig, val modelVersion: ProcessingTy
       (latestProcessVersion, maybeJson) match {
         case (Some(version), _) if isLastVersionContainsSameJson(version, maybeJson) && version.mainClass == maybeMainClass =>
           Right(None)
-        case (_, Some(json)) =>
+        case (versionOpt, Some(json)) =>
```

Comment:
Yes, we sort there by date but it should not make much difference since there is no case when higher value of `versionId` can be added with earlier date (or is it? :detective: ). Though we could make this change for consistency

Reply:
well... **if** time on server is set correctly, then it's probably not possible ;)